packages/core/src/components/Section/Section.tsx

```diff
@@ -0,0 +1,143 @@
+import { setId } from "@core/utils/setId";
+import { staticClasses, useClasses } from "./Section.styles";
+
+export { staticClasses as sectionClasses };
+
+export type HvSectionClasses = ExtractNames<typeof useClasses>;
+
+export interface HvSectionProps
+  extends Omit<HvBaseProps<HTMLDivElement>, "title"> {
+  /** The title of the section */
+  title: React.ReactNode;
+  /** Whether or not the section is expandable.  */
+  expandable?: boolean;
+  /** Whether the section is open or not, if this property is defined the accordion must be fully controlled. */
+  expanded?: boolean;
+  /** When uncontrolled, defines the initial expanded state. */
+  defaultExpanded?: boolean;
+  /** Section actions */
+  actions?: HvActionGeneric[];
+  /** Section action callback */
+  actionCallback?: HvActionsGenericProps["actionsCallback"];
+  /** Section maximum number of actions visible */
+  maxVisibleActions?: number;
+  /** Variant for the visible actions button */
+  actionsVariant?: HvButtonProps["variant"];
+  /** Section onExpand callback */
+  onToggle?: (id: string | undefined, open: boolean) => void;
+  /** Props to be passed to the expand button */
+  expandButtonProps?: HvButtonProps;
+  /** A Jss Object used to override or extend the styles applied to the empty state component. */
+  classes?: HvSectionClasses;
+}
+
+/**
+ * Sections allow grouping information on a page under the same topic.
+ */
+export const HvSection = (props: HvSectionProps) => {
+  const {
+    id,
+    classes: classesProp,
+    className,
+    title,
+    expandable,
+    expanded,
+    defaultExpanded = false,
+    actions,
+    actionCallback,
+    maxVisibleActions = 1,
+    actionsVariant = "secondarySubtle",
+    onToggle,
+    expandButtonProps,
+    children,
+    ...others
+  } = useDefaultProps("HvSection", props);
+
+  const { classes, cx } = useClasses(classesProp);
+
+  const [isOpen, setIsOpen] = useControlled(expanded, Boolean(defaultExpanded));
+
+  const elementId = useUniqueId(id, "hvSection");
+  const contentId = setId(elementId, "content");
+
+  const actsVisible = actions?.slice(0, maxVisibleActions);
+  const actsDropdown = actions?.slice(maxVisibleActions);
+
+  const showContent = expandable ? !!isOpen : true;
+
+  return (
+    <div id={elementId} className={cx(classes.root, className)} {...others}>
+      <div className={classes.header}>
+        {expandable && (
+          <div>
+            <HvButton
+              icon
+              onClick={() => {
+                setIsOpen((o) => !o);
+                onToggle?.(id, !isOpen);
+              }}
+              aria-expanded={isOpen}
+              aria-controls={contentId}
+              aria-label={isOpen ? "Collapse" : "Expand"}
+              {...expandButtonProps}
+            >
+              {isOpen ? <Up /> : <Down />}
+            </HvButton>
+          </div>
+        )}
+        <HvTypography variant="title4">{title}</HvTypography>
```

Comment:
I don't think we should assume the typography variant will always be `title4`.
Users will eventually ask for extra configuration. First the `variant`, then any typography props...
What about a sub-title?

We can eliminate the problem by simply not assuming anything.

```suggestion
        {title}
```